Dockerfile.ubuntu16.04.bats

```diff
@@ -25,3 +29,7 @@ RUN apt-get install -y net-tools iptables
 
 # Copy mock AWS CLI into the PATH
 COPY ./.circleci/aws-local.sh /usr/local/bin/aws
+
+# These have been added to resolve some encoding issues with the tests
```

Comment:
NIT: is there some GitHub issue or SO thread to explain the context here?

Reply:
Not a specific one, more like a combination of looking at some of the errors generated after upgrading to Python 3 and then some google-fu.

Here's an example of an error that happens after switching to Python 3 (but before these env variables are added):
```
bats_ubuntu1804_1  | not ok 57 aws_get_instance_tags empty
...
bats_ubuntu1804_1  | #     return codecs.ascii_decode(input, self.errors)[0]
bats_ubuntu1804_1  | # UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 13950: ordinal not in range(128)
```

That lead to finding threads like [this one](https://stackoverflow.com/questions/51026315/how-to-solve-unicodedecodeerror-in-python-3-6/51027262#51027262) which is where the environment variables have come from. I'll add some more details here and push that shortly.

Reply:
Perfect, thanks for adding that!